website/modules/@apostrophecms/form/lib/GoogleSheetsClient.js

```
@@ -0,0 +1,44 @@
+  }
+
+  async appendValues(range = APPEND_RANGE, values) {
+    try {
+      return await this.sheets.spreadsheets.values.append({
+        spreadsheetId: this.spreadsheetId,
+        range,
+        valueInputOption: 'RAW',
+        resource: { values },
+      });
+    } catch (err) {
+      throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)